alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineCliParserTest.scala

```diff
@@ -23,14 +23,22 @@ class AlertEngineCliParserTest extends FunSuite {
   /*
    *function to set environmental variables for testing
    */
-  def setEnv(key: String, value: String) = {
+  def setEnv(key: String, value: String): String = {
     val field = System.getenv().getClass.getDeclaredField("m")
     field.setAccessible(true)
     val map =
       field.get(System.getenv()).asInstanceOf[java.util.Map[java.lang.String, java.lang.String]]
     map.put(key, value)
   }
 
+  def unsetEnv(key: String): String = {
```

Comment:
Is this function used somewhere other than a test?

Reply:
No.